src/worker/piscina.js

```diff
@@ -6,12 +6,47 @@ if (isMainThread) {
     const { createConfig } = require('./config')
     module.exports = {
         makePiscina: (serverConfig) => {
-            const piscina = new Piscina(createConfig(serverConfig, __filename))
-            piscina.on('error', (error) => {
-                Sentry.captureException(error)
-                console.error('⚠️', 'Piscina worker thread error:\n', error)
-            })
-            return piscina
+            const config = createConfig(serverConfig, __filename)
```

Comment:
Can we remove piscina concurrency from this PR? 😋

Reply:
whoopie :D